core/services/registrysyncer/syncer.go

```diff
@@ -157,6 +157,7 @@ func (s *registrySyncer) syncLoop() {
 	s.lggr.Debug("starting initial sync with remote registry")
 	err := s.Sync(ctx, true)
 	if err != nil {
+		// TODO ks-461
```

Comment:
ks-461's are captured by the log alerting for EVM, and don't explicitly need custom message coverage for other chains